manifest.json

```diff
@@ -19,5 +25,15 @@
     "default_icon": "static/graffiome_logo_19.png",
     "default_title": "browser action",
     "default_popup": "src/browser_action/browser_action.html"
-  }
+  },
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
+      "js": [
+        "src/bower_components/jquery/dist/jquery.js",
+        "src/content-scripts/full-page-canvas.min.js",
```

Comment:
Is this a library or a minified version of the canvas.js? If the later how are you minifiing this?